openpifpaf/plugins/wholebody/wholebody.py

```diff
@@ -26,7 +26,7 @@
     pass
 
 
-class Wholebody(openpifpaf.datasets.DataModule):
+class Wholebody(openpifpaf.datasets.DataModule, openpifpaf.Configurable):
```

Comment:
`Configurable` class is still experimental. Can you leave a brief comment why this is necessary here?